nilearn/datasets/description/aal_SPM12.rst

```diff
@@ -1,11 +1,23 @@
-AAl atlas for SPM 12
+AAL atlas
+=========
 
 
 Notes
 -----
-This atlas is the result of an automated anatomical parcellation of the spatially normalized single-subject high-resolution T1 volume provided by the Montreal Neurological Institute (MNI) (D. L. Collins et al., 1998, Trans. Med. Imag. 17, 463-468, PubMed).
+This atlas is the result of an automated anatomical :term:`parcellation`
+of the spatially normalized single-subject high-resolution T1 volume
+provided by the Montreal Neurological Institute (MNI)
+(D. L. Collins et al., 1998, Trans. Med. Imag. 17, 463-468, PubMed).
```

Comment:
make it formally a citation  ?

Reply:
Actually there is a whole bunch in of other references mentioned that are not in any of our .bib files so I was planning to tackle those in a separate PR.

But let me know if you me to do it in this one.

Reply:
This PR is already pretty big. It's OK to me if you address this in another PR.
Best,